packages/services/f-analytics/src/services/_tests/analytics.service.test.js

```diff
@@ -16,55 +17,256 @@ describe('Analytic Service ::', () => {
     let store;
     let storeDispatchSpy;
     let service;
+    let req;
     let windowCopy;
     let windowSpy;
     let windowsPushSpy;
     let get;
-
-    const mockWindow = ({ winWidth = 667, winHeight = 375 } = {}) => {
-        windowsPushSpy = jest.fn();
-        windowCopy = { ...window };
+    let registerStoreModuleSpy;
+
+    const mockWindow = ({
+        winWidth = 667,
```

Comment:
Do you need to create these? You could just pass the value in directly to line 45 & 46 instead? that way you dont need the spy or to return anything